js/app/modules/card/media.js

```diff
@@ -58,12 +58,12 @@ const Media = new Module.Class({
             attributions.push(this.model.copyright_holder);
         this.set_label_or_hide(this._attribution_label, attributions.map((s) => {
             return s.split('\n')[0];
-        }).join(' - ').toUpperCase());
+        }).join(' - '));
         this._attribution_button.visible = this._attribution_label.visible;
         this._attribution_button.connect('clicked', this._show_license_in_external_viewer.bind(this));
-
-        this._separator.visible = this._attribution_label.visible &&
-                                  this._caption_label.visible;
+        if (this._caption_label.visible && this._attribution_label.visible) {
```

Comment:
Okay, I see what you mean; the only point of my comment was that it would be adding 2 classes to the module instead of 1, which seemed odd, but yeah, I agree that when looking at CSS It would look more intuitive. I'll change it!